hw/top_earlgrey/data/synth.xdc

```diff
@@ -0,0 +1,14 @@
+## Copyright lowRISC contributors (OpenTitan project).
+## Licensed under the Apache License, Version 2.0, see LICENSE for details.
+## SPDX-License-Identifier: Apache-2.0
+
+## Synthesis constraints
+
+# Prevent Vivado from combining these BRAMs, so the memories can be readily
+# spliced with updatemem.
+set_property KEEP_HIERARCHY TRUE [get_cells "top_earlgrey/u_flash_ctrl/u_eflash/u_flash/gen_generic.u_impl_generic/gen_prim_flash_banks[0].u_prim_flash_bank/gen_info_types[0].u_info_mem"]
+set_property KEEP_HIERARCHY TRUE [get_cells "top_earlgrey/u_flash_ctrl/u_eflash/u_flash/gen_generic.u_impl_generic/gen_prim_flash_banks[0].u_prim_flash_bank/gen_info_types[1].u_info_mem"]
```

Comment:
Another opportunity to deduplicate (probably not even using `eval`)?

Reply:
This actually gets added and removed again because of the changing ideas. If you'd like, I can go back in the history and get rid of its appearance altogether.